packages/checkout/sdk/src/types/smartCheckout.ts

```diff
@@ -50,6 +69,15 @@ export type BuyResultInsufficientFunds = {
   smartCheckoutResult: SmartCheckoutInsufficient
 };
 
+/**
+ * Represents the overrides available for {@link Checkout.buy}
+ * @property {boolean} waitFulfillmentSettlements
+ */
+export type BuyOverrides = {
+  /** If the buy should wait for the fulfillment transactions to settle */
```

Comment:
should we mention its true by default in the description

Reply:
I thought about this but decided not to as I didn't want the comment and code to become out of sync if that default ever changes. I guess the public facing endpoint isn't overly obvious though as it doesn't have a default. Would you like me to add it in here?

Reply:
The type itself cant have a default value but could the exposed function show there is a default value? i think we should communicate there is a default value, otherwise as an optional override from an integration perspective it might look like or be assumed that its false by default

Reply:
Lets just add this to the actual docs instead

Reply:
As discussed I'll update with description around defaults in the docs.